utils/rpm.mk

```diff
@@ -127,7 +127,7 @@ pwdr = $(subst ${PBENCHTOP},,${CURDIR})
 # inside the container as ${HOME}/rpmbuild (with no suffix), which is where
 # the build will put the RPM when invoked without a distro target.
 .PHONY: %-rpm
-%-rpm: spec srpm
+%-rpm: rpm-dirs
```

Comment:
That's a massive change from the previous logic. If I'm reading correctly, though, we're now just creating directories in the "outer" build while your `jenkins/run make` does the `rpmbuild` commands inside the sidecar container. Should there be a bit more context in the comment?

Reply:
The memories are foggy now, largely obliterated by the intervening trials with container-in-container attempts, but IIRC the previous logic was wrong -- I _think_ it resulted in building the SRPM _twice_ (per distro!).

Ideally, we would build the SRPM exactly once and then share it among all of the distro-specific binary RPMs, and, at some point, I'll experiment with that (there might be a nice trick for supporting it); however, mostly what the Makefile does now (at the top level) is kick off a bunch of sub-make's, each in a separate, appropriate container.  Each sub-make is self contained -- it builds its own SRPM and binary RPM -- and it puts the output in a file system which is mapped in from the host.  Inside the sidecar, they are all working identically, with the same output directory; outside, they each have a different OS in their container and a different target directory.  So, the top-level make only needs to create the output directories which will be mapped into the containers; the creation of the spec file and the SRPM is done in the sub-make, and so I've corrected the dependencies, here.

Reply:
Most of that response would probably be a useful comment to have in the file for future reference: decoding the structure and sequence of a makefile can be a twisty and mind-bending journey. 😆